plugins/sensors/upsmonpro_

```diff
@@ -0,0 +1,207 @@
+#!/usr/bin/perl
+
+=head1 NAME
+
+upsmonpro_ - Munin plugin to monitor Powercom UPS via UPSMON PRO program L<http://www.pcm.ru/support/soft/>
+
+=head1 INSTALLATION
+
+  /etc/munin/plugins/upsmonpro_load -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_status -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_temp -> /usr/share/munin/plugins/upsmonpro_
+  /etc/munin/plugins/upsmonpro_voltage -> /usr/share/munin/plugins/upsmonpro_
+
+=head1 CONFIGURATION
+
+Environment variables:
+
+  host - UPSMON PRO server host, default localhost
+  port - UPSMON PRO port, default 2601
```

Comment:
Listing the environment variables is correct and sufficient. But some (novice) users would probably appreciate something like the following for easier understanding:
```
Example configuration (optional):

  [upsmonpro_*]
  env.host localhost
  env.port 2601
```
What do you think?